File: java/MultiplicationTable.java
public class MultiplicationTable {
    public static void main(String[] args) {
        System.out.println("Multiplication Table v1.0");
        for (int i=1; i<=12; i++) {
            System.out.print('\n');
            for (int j=1; j<=12; j++) {
                System.out.println(i + " * " + j + " = " + (i*j));
            }
        }
    }
}

File: cs/multiplication-table.cs
using System;

namespace multiplication_table
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Multiplication Table (1-12)\n");
            for (int i = 1; i < 13; i++)
            {
                for (int j = 1; j < 13; j++)
                {
                    Console.WriteLine($"{i} * {j} = {i * j}");
                }
                Console.WriteLine("");
            }
        }
    }
}
